scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -178,7 +165,7 @@ public static BuildTargetResult BuildTestAssetPackages(BuildTargetContext c)
                 {
                     "netstandard1.5",
                     "netstandard1.3",
-                    "netstandardapp1.5"
+                    "netcoreapp1.0"
```

Comment:
Here's a classic case of having "why" we are doing things listed in comments.  I look at this code and I ask myself, "why are we doing this?", but there is no reason listed.